chromadb/db/impl/sqlite.py

```diff
@@ -34,21 +36,23 @@ def __exit__(
         return False
 
 
-class SqliteDB(MigratableDB, SqlEmbeddingsQueue):
+class SqliteDB(MigratableDB, SqlEmbeddingsQueue, SqlSysDB):
```

Comment:
It grows! So it seems like we will not have a way to split the code for these and all the implementations will live in this one class that extends every purpose? Can we just #region the code or similar so that its easier to navigate. (NIT)